.eslintrc.json

```diff
@@ -69,7 +69,7 @@
         "avoidEscape": true
       }
     ],
-    "linebreak-style": "error",
+    "linebreak-style": ["error", "windows"],
```

Comment:
Why would we want to do CRLF line endings?

Reply:
hello i didn't done it yet, please wait undrafted

Reply:
Sorry, then I'll let you continue and wait until it's ready for review. Thanks for working on it!